Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -368,12 +368,12 @@ open class ChatMessageListVC: _ViewController,
 
         let actionsController = components.messageActionsVC.init()
         actionsController.messageController = messageController
-        actionsController.channelConfig = dataSource?.channel(for: self)?.config
+        actionsController.channel = dataSource?.channel(for: self)
         actionsController.delegate = self
 
         let reactionsController: ChatMessageReactionsPickerVC? = {
             guard message.localState == nil else { return nil }
-            guard dataSource?.channel(for: self)?.config.reactionsEnabled == true else {
+            guard dataSource?.channel(for: self)?.ownCapabilities.contains(.sendReaction) == true else {
```

Comment:
I think this validates the comment on having the properties as extension of ChatChannel